packages/client/components/ReflectionEditorWrapper.tsx

```diff
@@ -169,7 +169,11 @@ class ReflectionEditorWrapper extends PureComponent<Props> {
         this.removeModal()
       } else {
         const el = this.props.editorRef.current
-        el && el.blur()
+        // add to callback queue so we can check activeElement and
+        // determine whether modal should close in expandedReflectionStack
+        setTimeout(() => {
+          el && el.blur()
```

Comment:
I noticed that in prod, when editing a reflection in the expanded reflection stack and hitting esc, the modal does actually close. This is because we've got a keydown event handler in [ExpandedReflectionStack](https://github.com/ParabolInc/parabol/blob/6cfe9aac9dd0ae342e1310b3edb45845516d389b/packages/client/components/RetroReflectPhase/ExpandedReflectionStack.tsx#L100) that closes the modal and we're passing and we're preventing the default close behaviour in usePortal [here](https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/client/hooks/useExpandedReflections.ts#L40).

I'm not in love with the solution, but one option is using this setTimeout so that we can see where the activeElement is in `ExpandedReflectionStack`. If it's focused on editing the card, just blur. Otherwise, close the modal